tests/subsys/pm/device_wakeup_api/src/main.c

```diff
@@ -16,44 +16,47 @@ static const struct device *dev;
 static uint8_t sleep_count;
 
 
-void pm_power_state_set(struct pm_state_info info)
+void pm_power_state_set(enum pm_state state, uint8_t substate_id)
 {
-	ARG_UNUSED(info);
+	ARG_UNUSED(substate_id);
 
-	enum pm_device_state state;
+	enum pm_device_state p_state;
```

Comment:
since it represents device state maybe:

```suggestion
	enum pm_device_state dev_state;
```